ee_tests/README.md

```diff
@@ -66,7 +89,7 @@ NODE_DEBUG=true npm start
 DEBUG=true npm start
 ```
 
-By default, Google Chrome browser is used by EE tests via Selenium. If you want
+By default, Google Chrome browser is used by E2E tests via Selenium. If you want
```

Comment:
Remove reference to Firefox, since nobody has actually tired that in a very long time.